src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -73,7 +75,9 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Chat.Prefixes.Nonbender", "[Nonbender]");
 
 			config.addDefault("Board.Title", "&lAbilities");
-			config.addDefault("Board.SelectionPrefix", ">  &r");
+			config.addDefault("Board.Prefix.Text", "> ");
+			config.addDefault("Board.Prefix.SelectedColor", ChatColor.WHITE.toString());

```

Comment:
In the future, we should consider giving people a customized version of the bending board that they create with a given syntax and we just parse for colors, like Placeholder API or something.

Reply:
mm could be a good idea, definitely for the future though